.circleci/config.yml

```diff
@@ -97,13 +97,14 @@ jobs:
       - run: apk update && apk add git make curl tar
       - checkout
       - gomod
-      - setup_remote_docker
+      - setup_remote_docker:
```

Comment:
Our circleci `build_image` test is failing on this PR. Is this behaviour expected?

Reply:
It actually worked for like 5 previous runs, but failed this time. Now I don't know how to fix it. This was supposed to work.